ref/feature/examples/LDAP-example.adoc

```diff
@@ -0,0 +1,75 @@
+
+== Examples
+
+=== Simple LDAP registry configuration example
+
+Configure the LDAP user registry by adding the `<ldapRegistry>` element to your `server.xml` file.
+Then, specify the base Distinguished Name, the host, LDAP type, port number and realm attributes.
+Customize the group and user filters that are used to perform searches by adding the `<idsFilters>` sub-element.
+Use the `%v` variable for attribute value assertions to be replaced by the login name.
```

Comment:
The %v is replaced by the login name for users, and group name for groups. I might also specify that the idsFilters is specific to the type (IBM) and that there are others depending on the 'ldapType' specified.